src/Common/src/Interop/Windows/kernel32/Interop.QueryFullProcessImageName.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+using System.Text;
+
+internal partial class Interop
+{
+    internal partial class Kernel32
+    {
+        [DllImport("kernel32.dll", SetLastError = true)]
+        internal static extern bool QueryFullProcessImageName([In]IntPtr hProcess, [In]int dwFlags, [Out]StringBuilder lpExeName, ref int lpdwSize);
```

Comment:
In most of our P/Invokes, I believe we force them to use the 64-bit version, e.g. adding 'W' to the name, specifying CharSet=Unicode, etc.  Do we want to do that here?